### ISSUE TITLE
Racket Fury: Table Tennis VR (555050)

### ISSUE THREAD

**Issue by mimattr** [ValveSoftware/Proton#4393]
# Compatibility Report
- Name of the game with compatibility issues: **Racket Fury: Table Tennis VR**
- Steam AppID of the game: **555050**

## System Information
- GPU: **RTX 2060**
- Driver/LLVM version: **NVIDIA 455.38**
- Kernel version: **5.4.0-54-generic**
- [Link to full system information report as Gist](https://gist.github.com/mimattr/b0b7f46366e31e254460b9593ce87e51)
- Proton version: **5.13-2**
- SteamVR version: **1.15.10**

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-555050.zip](https://github.com/ValveSoftware/Proton/files/5561145/steam-555050.zip)

## Symptoms
The application appears to work as expected except substantial tracking latency on input devices (tested with Valve Index Controllers)

## Reproduction
1. Select the title within your `Library` using the Steam client and press the `Play` button

**Comment by kisak-valve** [comment]
Hello @mimattr, just to confirm, you're seeing the same late tracking behavior with Proton logging disabled?

**Comment by mimattr** [comment]
> you're seeing the same late tracking behavior with Proton logging disabled?

@kisak-valve Correct, almost feels like the controllers are moving at a lower framerate than the HMD.
